recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "arcticdb" %}
-{% set version = "4.3.2rc2" %}
+{% set version = "4.4.0rc1" %}
```

Comment:
Nitpick: usually `rc` starts at 0, and I think `4.4.0rc0` should have been picked; but this is of minor importance I think.

Reply:
I had created an [4.4.0rc0 tag](https://github.com/man-group/ArcticDB/tags) but we found a problem with it before adding it to the feedstock. rc1 should have that issue fixed. I guess I could have also moved the rc0 tag to the new commit but probably not worth doing it now.

Reply:
OK it makes sense if `rc0` was tagged before (I did not check its existence).